internal/core/docker_rootless_test.go

```diff
@@ -178,14 +178,8 @@ func TestRootlessDockerSocketPath(t *testing.T) {
 		setupRootlessNotFound(t)
 
 		socketPath, err := rootlessDockerSocketPath(context.Background())
-		require.ErrorIs(t, err, ErrRootlessDockerNotFound)
+		require.Error(t, err)
```

Comment:
suggestion: shouldnt this be:
```suggestion
		require.ErrorIs(t, err, ErrRootlessDockerNotFoundXDGRuntimeDir)
```